### ISSUE TITLE
Segmentation fault: 11

### ISSUE THREAD

**Issue by jasverj** [JuliaLang/julia#35005]
I recently got a new iMac 2019 with a 3.6 GHz 8-Core Intel Core i9 and 40 GB 2400 MHz DDR4 Memory (2 x 4 GB + 2 x 16 GB). I am running the LTS version of Julia (v1.0.5) using Atom and I have added the following packages: CSV v0.5.14, BSON v0.2.4, Revise v2.2.2, Distributions v0.21.5, Juno v0.7.2, Atom v0.12.7, StatsBase v0.32.0, MultivariateStats v0.7.0, Tables v0.2.11, Flux v0.9.0, PyPlot v2.8.2, and DataFrames v0.19.4. Note that these are not the most updated version of the packages; I have these exact versions installed on another machine (MacBook Pro 2013 2.3 GHz Quad-core Intel Core i7 and 16 GB 1600 MHz DDR3 Memory), where no Seg fault occurs. I executed the following code:

```
    using Distributed, Flux, DataFrames, Statistics, LinearAlgebra, StatsBase, MultivariateStats, Distributions, CSV, Revise
    addprocs(length(SYS.cpu_info()))
    @everywhere basedir = homedir()
    @everywhere cd(basedir * "/Desktop/CYCLOPS")
```

At this point I usually get the following:

> signal (11): Segmentation fault: 11
> in expression starting at no file:0
> _platform_strcmp at /usr/lib/system/libsystem_platform.dylib (unknown line)
> Allocations: 140732868 (Pool: 140702277; Big: 30591); GC: 306

This doesn't happen only when executing lines of code. It happens randomly but also when I make changes to different lines of code, _without_ executing that line of code. I don't know if this related to the issue related to DataFrames causing Seg fault in julia LTS, or if this may be an issue with Juno and/or Atom. Thank you for any help!

**Comment by PallHaraldsson** [comment]
While 1.0.5 is the latest LTS version (and good to know this), could you check on the latest version? For me it works (while I made changes that shouldn't matter).

FYI, you need:
Distributed.addprocs(length(Base.Sys.cpu_info()))

and better:

@everywhere cd(joinpath(basedir, "Desktop", "CYCLOPS") # more portable

At first I thought you might be running out of memory, while these lines of code seem simple enough, and shouldn't allocate much? However at least on Linux it doesn't take much, and could be some other program responsible for filling up memory... OOM is a funny/bad business. I see you're on a Mac and not sure if it's similar.

[I reported some crash myself for the LTS version, which works on newer. That issue is being ignored. I suspect since it's working in later (and for an obscure package). What most should be using anyway. And if I recall, 1.4, or 1.5 will likely be the new LTS, soon I think. I guess 1.x will then be dropped and old issues on it.]

**Comment by JustinLawrenceMS** [comment]
Did you fix this?  I'm getting this error when trying to access TableView.showtable() from a user defined function.  New to Julia so I'm probably way off the mark.   Also using a Mac.

**Comment by JustinLawrenceMS** [comment]
I appear to have solved my issue by moving the database connection outside of the function call.

**Comment by PallHaraldsson** [comment]
In general, I would run the latest version (and/or nightly version), e.g. just released v1.5.3 (Nov 9, 2020) Julia version like this:

``
julia --check-bounds=yes
```

in case you get segfaults, to try to find the cause. Julia 1.6 will be the next LTS, not right away (it will first be released as non-LTS very soon), see here: https://discourse.julialang.org/t/what-are-the-guarantees-of-lts-julia-versions/39572/11?u=palli
